### ISSUE TITLE
Postprocessor doesn't convert CJS to goog module properly in ES5 mode

### ISSUE THREAD

**Issue by vikerman** [angular/tsickle#162]
This is a particular case where in code there is an export * before import from the same module
```
export * from './random';
import {getRandom} from './random';
```
When TS outputs ES5+CJS mode it looks like

```
"use strict";
function __export(m) {
    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
}
__export(require('./random'));
var random_2 = require('./random');
```
When the tsickle Postprocessor runs on this, it outputs

```
goog.module('codelab.main');var module = module || {id: 'codelab/main'};
function __export(m) {
    for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
}
__export(goog.require('google3.javascript.typescript.examples.codelab.random'));
var random_2 = *;  <------------- Oops!!
```

The problem is here https://github.com/angular/tsickle/blob/master/src/tsickle.ts#L1179

It outputs working code(by generating two separate goog.require for the export and the import) if the above line is changed to 

```
if (this.moduleVariables.hasOwnProperty(modName) && this.moduleVariables[modNam] !== '*') {
```

**Comment by martine** [comment]
It sounds like, due to https://github.com/angular/tsickle/issues/118 , we cannot just emit
```js
var random_2 = goog.require('google3.javascript.typescript.examples.codelab.random');
```
because that would cause a duplicate import.  Still looking...

**Comment by martine** [comment]
Fixed in 31f9b4220c8124ab34dde5a6800ed228ae476d2c
